docs/data/sdks/ios-swift/ampli.md

```diff
@@ -0,0 +1,205 @@
+    Ampli.instance.songPlayed(SongPlayed(songId: 'song-1');
+    Ampli.instance.track(SongFavorited(songId: 'song-2');
+    ```
+
+7. [Flush events before application exit](#flush)
+
+    ```swift
+    Ampli.instance.flush()
+    ```
+
+8. [Verify implementation status with CLI](#status)
+
+    ```shell
+    ampli status [--update]
+    ```
+
+## Installation
+
+### Install the Amplitude SDK
+
+If you haven't already, install the core Amplitude SDK dependencies.
+
+--8<-- "includes/sdk-swift/swift-install-dependencies.md"
+
+--8<-- "includes/ampli/cli-install-simple.md"
+
+## API
+
+### Load
+
+Initialize Ampli in your code. The `load()` method accepts configuration option arguments:
+
+```swift
+Ampli.instance.load(LoadOptions(
+  environment: AmpliEnvironment.Production
+));
+```
+
+| Arg | Description |
+|-|-|
+|`LoadOptions`| Required. Specifies configuration options for the Ampli Wrapper.|
+|`environment`| Required. String. Specifies the environment the Ampli Wrapper is running in. For example,  `production` or `development`. Create, rename, and manage environments in Amplitude Data.<br /><br />Environment determines which API token is used when sending events.<br /><br />If a `client.apiKey` or `client.instance` is provided, `environment` is ignored, and can be omitted.|
+|`disabled`|Optional. Specifies whether the Ampli Wrapper does any work. When true, all calls to the Ampli Wrapper are no-ops. Useful in local or development environments.|
+|`instance`|Optional. Specifies an Amplitude instance. By default Ampli creates an instance for you.|
+|`apiKey`|Optional. Specifies an API Key. This option overrides the default, which is the API Key configured in your tracking plan.|
+
+### Identify
+
+Call `identify()` to identify a user in your app and associate all future events with their identity, or to set their properties.
+
+Just as the Ampli Wrapper creates types for events and their properties, it creates types for user properties.
+
+The `identify()` function accepts an optional `userId`, optional user properties, and optional `options`.
+
+For example your tracking plan contains a user property called `userProp`. The property's type is a string.
+
+```swift
+Ampli.instance.identify("userID", Identify(
+  userProp: "A trait associated with this user"
+));
+```
+
+The options argument allows you to pass [Amplitude fields](https://developers.amplitude.com/docs/http-api-v2#keys-for-the-event-argument) for this call, such as `deviceId`.
+
+```swift
+Ampli.instance.identify("userID", Identify(deviceID: "my_device_id")
+```
+
+### Group
+
+!!! note
+    This feature is available for Growth customers who have purchased the [Accounts add-on](https://help.amplitude.com/hc/en-us/articles/115001765532).
+
+Call `setGroup()` to associate a user with their group (for example, their department or company). The `setGroup()` function accepts a required `groupType`, and `groupName`.
+
+```swift
+Ampli.instance.client.setGroup("groupType", "groupName")
```

Comment:
It looks like argument labels should be used here:
https://github.com/amplitude/Amplitude-Swift/blob/main/Sources/Amplitude/Amplitude.swift#L130

Reply:
Updated.